utils/generate-commands-json.py

```diff
@@ -133,4 +133,10 @@ def convert_entry_to_objects_array(cmd, docs):
         name = list(cmd.keys())[0]
         payload[name] = cmd[name]
 
-    print(json.dumps(payload, indent=4))
+    # Print the final JSON output. If the output is piped and the pipe is closed (e.g., by 'less' or 'head'),
+    # catch BrokenPipeError to prevent a traceback and exit gracefully.
+    try:
+        print(json.dumps(payload, indent=4))
+    except BrokenPipeError:
+        sys.stderr.close()
+        sys.exit(0)
```

Comment:
why do we need this?

Reply:
As I put in the comments, it's just to exit gracefully if you try `generate-commands-json.py | less`.